File: 3606-minimum-element-after-replacement-with-digit-sum/minimum-element-after-replacement-with-digit-sum.js
var minElement = function (nums) {
    let ans = 37;
    for (const num of nums) {
        let dig = 0
        let n = num
        while (n) {
            dig += n % 10
            n = Math.floor(n / 10)
        }
        ans = Math.min(ans, dig)
    }
    return ans
};

File: 3606-minimum-element-after-replacement-with-digit-sum/minimum-element-after-replacement-with-digit-sum.cpp
class Solution {
public:
    int minElement(vector<int>& nums) {
        int ans = INT_MAX;
        for (int num : nums) {
            int dig = 0;
            while (num) {
                dig += num % 10;
                num /= 10;
            }
            ans = min(ans, dig);
        }
        return ans;
    }
};